srcs/layouts/guj_phonetic_in.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<keyboard name="ગુજરાતી ફોનેટિક - Gujarati Phonetic" script="gujarati">
+  <row>
```

Comment:
The top row doesn't look good in combination to the built-in number row (that can be enabled in the settings). Do you think every users of this layout will want to have this row ?

Also, the layout is very high due to top row. It takes almost half of the screen on my device. You can make the top row a bit smaller with `height="0.75"` (like the built-in number row).

Reply:
![Screenshot_2024-02-18-22-29-59-773_com.github.android.jpg](https://github.com/Julow/Unexpected-Keyboard/assets/41276813/2664b3b8-7af7-44e6-bea3-27f337e77e98)

we can do something like this to remove the top row. [**as you see in the Screen Shot, numbers are shown properly in keyboard but output is not in devanagari. I feel like there is a bug. It devanagari numbers work fine in normal view, just this number view is not working proper]

Reply:
The devanagari digits are specified here: https://github.com/Julow/Unexpected-Keyboard/blob/master/srcs/juloo.keyboard2/KeyModifier.java#L1060
Are they wrong ?

To change the numpad digits when your layout is visible, you'd need to add a new similar map and call it from here: https://github.com/Julow/Unexpected-Keyboard/blob/master/srcs/juloo.keyboard2/KeyModifier.java#L100

Reply:
The first screen shot just shows the layout in Devanagari(or any other language), but when typing, it puts output as english numbers. I think it is using values from https://github.com/Julow/Unexpected-Keyboard/blob/master/res/xml/numpad.xml .  

I tried adding Gujarati number in the java file you suggested, but had same issue as mentioned above.

Reply:
You are right, there's a serious bug in the numpad. I fixed it in 8c7559d and fixed the number row in 8b5d971.

You can now remove the number row from your layout and enable the number row in the settings.

Note that the layout must also contain the numerals for people that don't use the number row option.